ui/apps/platform/cypress.config.js

```diff
@@ -7,7 +7,6 @@
  */
 
 module.exports = {
-    blockHosts: ['*.*'], // Browser options
```

Comment:
We do not want to block hosts, as we need the tests to connect to a URL that isn't known until runtime.